src/components/OntologyBrowser/Test/OntologyModal.test.tsx

```diff
@@ -150,3 +150,104 @@ describe('Ontology Modal', () => {
     expect(wrapperInstanceCombine.instance().getKeysFromSqon().sort()).toEqual([]);
   });
 });
+
+describe('updateSqons method render updateSqon and', () => {
+  it('not alter initial input Sqon', () => {
+    const initialSqon = {
+      op: 'and',
+      content: [{ op: 'in', content: { field: 'gender', value: ['Male'] } }],
+    };
+
+    const refInitalSqon = JSON.parse(JSON.stringify(initialSqon));
```

Comment:
I would avoid this line and do:

expect({
      op: 'and',
      content: [{ op: 'in', content: { field: 'gender', value: ['Male'] } }],
    }).toEqual(initialSqon);

same thing for the other tests (I think it should be avoided JSON.parse(JSON.stringify(initialSqon));)